plugins/py3test.py

```diff
@@ -0,0 +1,92 @@
+from util import hook
+import os
+import sys
+
+# this absolute meme of a plugin outputs irc lines to an unix named pipe,
+# which means other software can read irc and interact with taiga
+
+# WARNING this plugin cannot be reloaded
+# IT WILL BREAK IF ITS RELOADED
+
+# named pipes go through memory and they never touch the network or the hard drive,
+# so data transfers should be pretty much instant with no cpu usage
+
+# TODO find a better way to encode taiga data in a line
+# just went with 'unicode unit separator' because its unlikely to be in an irc message

```

Comment:
NOOOOOOOOOOOOOOOOOOOOOOOOoooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo

Reply:
we should probably use a Tag-Length-Value or some other common document format, like god forbid JSON. hold up on this. let me fuck with it locally.

Reply:
i think i'm going to leave this thing to the other branch, maybe even add the stdin/stdout thing for adding neo8ball-like plugin support

Reply:
### **dude lets use xml**